lib/git-whistles/youtrack/ticket.rb

```diff
@@ -0,0 +1,60 @@
+module Git::Whistles
+  module Youtrack
+    class Ticket
+      class << self
+        def build_from_remote(ticket_hash)
+          return nil if ticket_hash.nil?
+          return nil if issue_not_found? ticket_hash
+
+          self.new.tap do |ticket|
+            ticket.title = title_from ticket_hash
+            ticket.id = id_from ticket_hash
+            ticket.project = project_from ticket_hash
+            ticket.description = description_from ticket_hash
+          end
+        end
+
+        def title_from ticket_hash
```

Comment:
Comments for all the following code:
- Use parenthesis for arguments
- Worth to cache value?
- Extra white space on the select method call
- Please have the dots starting on each line
- Dig will require > 2.2 I'm not sure how many users we have on old versions of ruby... but well lets leave it as it is for now